pypesto/hierarchical/optimal_scaling_approach/optimal_scaling_parameter.py

```diff
@@ -0,0 +1,52 @@
+import logging
```

Comment:
Ideally add a 1 line description at the beginning of each file what it is about.

sth like ``"""Definition of an optimal scaling parameter class."""``